modules/Emails/controller.php

```diff
@@ -324,23 +333,34 @@ public function action_GetFolders()
         $this->view = 'ajax';
     }
 
+
+    public function action_DisplayDetailView()
+    {
+        $emails = BeanFactory::getBean("Emails");
+        $result = $emails->get_full_list("", "uid = {$_REQUEST['uid']}");
```

Comment:
can you use the db->quote method on the request variable before use it in the sql queries, also make sure this place the "uid" could be only number or strings also acceptable because if it could be a string you should use quotes